src/components/transport_manager/include/transport_manager/bluetooth/bluetooth_connection_factory.h

```diff
@@ -90,6 +93,15 @@ class BluetoothConnectionFactory : public ServerConnectionFactory {
   virtual ~BluetoothConnectionFactory();
  private:
   TransportAdapterController* controller_;
+  ConnectionHolder connections_;
+
+  /**
+   * @brief Destructor.
+   *
+   * Checks status of the container connections using
+   * the transport adapter controller.
+   */
+  void removeFinalizedConnections();
```

Comment:
@vlantonov Wrong method name. Should start from capital letter. Please use [Google Coding Style](https://google.github.io/styleguide/cppguide.html#Function_Names)